client/my-sites/email/email-management/email-home.tsx

```diff
@@ -96,6 +99,7 @@ const EmailHome = ( props: EmailManagementHomeProps ) => {
 		return canCurrentUser( state, selectedSite.ID, 'manage_options' );
 	} );
 	const hasSitesLoaded = useSelector( hasLoadedSites );
+	const isAllDomainManagementEnabled = context === 'domains';
```

Comment:
Sure, good catch. The naming is a leftover from the previous approach in which we relied on a feature flag instead of context.

I'll update it. Thanks!!!